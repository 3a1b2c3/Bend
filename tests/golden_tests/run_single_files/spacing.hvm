(Main) = λa (λa
a
a a)