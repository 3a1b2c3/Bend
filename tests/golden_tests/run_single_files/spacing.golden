dup b c = a; Î»a (b c)